woocommerce/woocommerce-android
# Adds tracking to site visibility editing

Closes: #13222 


### Description

Adds tracking to edit site's visibility feature. 

### Steps to reproduce


### Testing information
1. Open site picker screen and check the following log is shown once site's are loaded: `🔵 Tracked: woocommerceandroid_site_picker_edit_button_shown`
2. Tap on `Edit Stores` cta an verify the following is shown `🔵 Tracked: woocommerceandroid_site_picker_edit_button_tapped,`
3. Edit the selected sites and click `Save`. Check the following is tracked: `🔵 Tracked: woocommerceandroid_site_picker_list_save_button_tapped, Properties: {"hidden_site_count":2`

**Error tracking**

1. Go to settings > Developer options > Api faker
2. Add the following endpoint and enable API faker: 

<img width="200"  src="https://github.com/user-attachments/assets/75475c2f-6eea-4368-bd14-1ccc5c9d2618"> 

3. Check the following event is tracked: `🔵 Tracked: woocommerceandroid_site_picker_list_saving_failure, Properties: {"error":"ApiError(message=)"`

### The tests that have been performed


The above.

### Images/gif



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
